tools/extra/fpgadiag/mactest.py

```diff
@@ -31,11 +31,11 @@
 import binascii
 import sys
 import glob
-from common import FpgaFinder, exception_quit
-from common import COMMON, convert_argument_str2hex
+from common import FpgaFinder, exception_quit, COMMON, hexint
+
+SYSF_IF = '/sys/class/net'
+DIVIDE = '-' * 80
```

Comment:
Lines 85 and 130 also need to be updated.

Reply:
Fixed